packages/components/src/utils/config-values.js

```diff
@@ -50,6 +51,8 @@ export default {
 	controlHeightLarge: `calc( ${ CONTROL_HEIGHT } * 1.2 )`,
 	controlHeightSmall: `calc( ${ CONTROL_HEIGHT } * 0.8 )`,
 	controlHeightXSmall: `calc( ${ CONTROL_HEIGHT } * 0.6 )`,
+	controlInnerControlTextColor: COLORS.darkGray,
+	controlTextColor: COLORS.darkGray,
```

Comment:
Same as above. These are fairy arbitrary and are probably wrong, or belong in COLORS. 

I'm not aware of any system to follow 🤷